sbom-ingestion.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: Ingesting SBOMs into GUAC
+permalink: /ingesting-sboms/
+---
+
+### Overview
+
+Software Bill of Materials (SBOM) ingestion is essential in GUAC (Graph for Understanding Artifact Composition) to help track and analyze dependencies, vulnerabilities, and software supply chain metadata. This documentation explains how to ingest SBOMs in GUAC, covering:
+- **Supported formats and limitations**
+- **Ingestion methods**
+- **Daemon-mode ingestion**
+- **Manual vs. automatic ingestion** 
+
+---
+
+### Supported SBOM Formats
+
+GUAC supports a variety of SBOM formats, making it compatible with several tools and standards:
+- **SPDX**: A widely used open standard for software package metadata.
+- **CycloneDX**: An SBOM specification built for security use cases.
+
+---
+
+### Ingestion Methods
+
+1. **Manual Ingestion**:
+   - Use the following command to ingest an SBOM file into GUAC for analysis and tracking dependencies:
+   - Example:
+     ```bash
+     guacone collect files my-sbom.spdx.json
+     ```
+   - This method is ideal when working with specific files or testing new SBOMs locally.
+
+2. **Daemon-Mode Ingestion (Polling Collectors)**:
+   - When configured, GUAC operates in **daemon mode**, using collectors (like GCS) to poll for new SBOMs at regular intervals.
+   - **How to Enable Daemon Mode**:
+     - Set up a configuration file specifying the collector sources.
+     - Example of a collector config:
+       ```yaml
+       collectors:
+         - type: gcs
+           bucket: my-sbom-bucket
+           pollingInterval: 5m
+       ```
+   - Once activated, GUAC will **automatically fetch and ingest** SBOMs from the specified source.
```

Comment:
What's your source for that example collector config? It doesn't line up with the comment I left in the last round of review.

Reply:
Okay, but now we're not explaining what to do.